config/locales/en/course/assessment/assessments.yml

```diff
@@ -39,6 +39,7 @@ en:
             text_response: 'Text Response'
             file_upload: 'File Upload'
             programming: 'Programming'
+            scribing: 'Scribing'
```

Comment:
That is for the create new question link in the assessment page which is not committed yet so we do not expose the scribing feature. Should I removed this first?